schemas/pioneer-citp-news-disinfo-two/rallymeasurements/rallymeasurements.2.schema.json

```diff
@@ -0,0 +1,265 @@
+{
+  "$comment": "Information about this study is available at https://github.com/citp/news-disinformation-study",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "mozPipelineMetadata": {
+    "bq_dataset_family": "pioneer_citp_news_disinfo_two",
+    "bq_metadata_format": "pioneer",
+    "bq_table": "rallymeasurements_v2",
+    "expiration_policy": {
+      "delete_after_days": 180
+    }
+  },
+  "properties": {
+    "newsAndDisinfo.linkExposure": {
```

Comment:
Hm, we've been formatting it like that from the beginning, so I guess it's typical for this schema :) By the time it gets to the analysis environment, all of these names have been normalized from camel case and `.`s to snake, which is working fine for us. Is there a reason to write it that way in the schema (instead of continuing to let the pipeline setup take care of it)?

Reply:
No particular reason, other than possibly some confusion (since `.` is often a path separator, on both JSON and BQ). More confusion would be caused by changing the formatting, so let's keep as-is.